GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -159,37 +180,71 @@ private void applicationFontChangeButton_Click(object sender, EventArgs e)
 
             if (result == DialogResult.OK || result == DialogResult.Yes)
             {
-                SetCurrentApplicationFont(applicationDialog.Font);
+                _applicationFont = applicationDialog.Font;
+                SetCurrentApplicationFont(_applicationFont);
+            }
+        }
+
+        private void chkApplicationFontSystemDefault_CheckedChanged(object sender, EventArgs e)
+        {
+            if (((CheckBox)sender).Checked)
```

Comment:
This is rather unsafe. Whilst it is against guidelines, some devs like invoking handlers directly passing arbitrary parameters (e.g. null).
Please change it to `if (chkApplicationFontSystemDefault.Checked)`.

Same applies to `chkCommitFontSystemDefault_CheckedChanged` and `lvTranslations_ItemActivate`

Reply:
Done, though I feel that's the point of the `sender` parameter. What about
```
var checkBox = sender as CheckBox;
if (checkBox != null)
{
    if (checkBox.Checked) { ... } else { ... }
}```?

Reply:
You can do that too. This pattern is used when the same handler is attached to multiple controls and there is a need to distinguish between them. 
In this current scenario you can reference the control directly.

Reply:
There's one more instance in FormChooseTranslation.cs in `lvTranslations_ItemActivate`